tensorboard/program.py

```diff
@@ -237,6 +240,37 @@ def launch(self):
     thread.start()
     return server.get_url()
 
+  def _install_signal_handler(self, signal_name):
+    """Set a signal handler to gracefully exit on the given signal.
+
+    When this process receives the given signal, it will run `atexit`
+    handlers and then exit with `0`.
+
+    If the current platform does not support a signal by the given name,
+    no action is taken and no error is raised.
+
+    Args:
+      signal_name: The name of a signal constant that may exist on the
+      `signal` module, like "SIGTERM" or "SIGQUIT".
+    """
+    signum = getattr(signal, signal_name, None)
+    if signum is None:
+      # Doesn't exist on this platform (e.g., SIGQUIT on Windows).
```

Comment:
I'd prefer to have the caller do this by hand, since it's a simple enough check and that way it's clear at the callsite that it won't always run.  That said, if you don't handle SIGQUIT then there's no need anyway.

Reply:
Yep; I’ll remove `SIGQUIT`, which removes the need for this wart.